codeFellowship/codeFellowship/src/main/resources/templates/user-info.html

```diff
@@ -1,23 +1,26 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<link rel="stylesheet" type="text/css" href="/css/styles.css">
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
 </head>
 <body>
 <h3 th:if="${username} != null" th:text="${'Current browsing user: ' + username}"></h3>
-
+<div th:replace="/fragments/error-fragment :: siteuser-error"></div>
```

Comment:
Make sure you've defined a fragment called "siteuser-error"

Reply:
thanks for the help Lucy! I will go back and look over this when I have time. There is a ton of code and I have found it is easy to get lost with all of the moving pieces.